NitroxServer/Communication/Packets/Processors/ChatMessageProcessor.cs

```diff
@@ -15,8 +15,13 @@ public ChatMessageProcessor(PlayerManager playerManager)
 
         public override void Process(ChatMessage packet, Player player)
         {
-            Log.Info(string.Format("<{0}>: {1}", player.Name, packet.Text));
-            playerManager.SendPacketToOtherPlayers(packet, player);
+            if (player.Muted)
+            {
+                player.SendPacket(new ChatMessage(ChatMessage.SERVER_ID, "You're currently muted"));
+                return;
+            }
+            Log.Info($"<{player.Name}>: {packet.Text}");
+            playerManager.SendPacketToAllPlayers(packet);
```

Comment:
```suggestion
            playerManager.SendPacketToOtherPlayers(packet, player);
```

Reply:
I think this is intentional as this is the only way for the client to know if he is (still) muted

Reply:
yes this is to lend the authority to the server